src/ngraph/runtime/cpu/builder/dot.cpp

```diff
@@ -251,7 +256,7 @@ namespace ngraph
 
                 auto functor =
                     [&, kernel, arg0_shape, arg1_shape, result_shape, reduction_axes_count](
-                        CPURuntimeContext* ctx) {
+                        CPURuntimeContext* ctx, CPUExecutionContext* ectx) {
                         kernel(arg0_tensor,
```

Comment:
This kernel() call doesn't have arena. whereas other kernel calls in this file do.